### PR TITLE
fix(messages): processes

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: messages.md [hunk 1]
```diff
@@ -634,48 +634,25 @@ The server must:
 
 ## Processes
 
-### Securing process
+### Presigning process  
+
+#### Revocation signing step
```
Comment (darosior) [inline messages.md:639]: nit: "Revocation transactions signing step" ?

File: messages.md [hunk 2]
```diff
@@ -634,48 +634,25 @@ The server must:
...
+   | +-sig cancel-------> |
+   |                      |
+   | +--get_sigs--------> |
+   |                      +
    |
    |                   watchtower
    |                       +
+   | +--sig emer---------> |
+   | <--------sig_ack----+ |
+   |                       |
```
Comment (darosior) [inline messages.md:655]: Needs the `sig cancel` too i guess

**Review by darosior — COMMENTED** [review]
Looks good, two small things :)

File: messages.md (hunk 2 above)
Comment (edouardparis) [reply messages.md:655]: It is present, check line 659

Comment (darosior) [reply messages.md:655]: Not sent to the WT

**Comment by darosior** [comment]
ACK  c42f125

**Pull request merged**